src/views/VideoView/VideoView.tsx

```diff
@@ -70,6 +82,30 @@ const VideoView: React.FC<RouteComponentProps> = () => {
     })
   }, [addVideoView, videoID])
 
+  // Save the video timestamp
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why do we disable that rule? The only missing dependency here is `updateWatchedVideos` I think? And this one should be stable so no trouble in including it?